puppeteer-test/src/tests/navigation/navigationmenu.spec.ts

```diff
@@ -0,0 +1,68 @@
+import GoogleLoginPage from '../../pages/google-login';
```

Comment:
It may be more maintainable to use absolute paths rather than relative paths, if possible. This is what our UI codebase uses in most cases... if it's not possible here for some reason, I'd love to know why (and quickly see if we can fix it).